server/scripts/clear-data.js

```diff
@@ -0,0 +1,19 @@
+require('dotenv').config({ path: '../.env' });
+const { dbClient } = require('../db');
+const { databaseCollections } = require('../db/schema');
+
+(async () => {
+  if (require.main === module) {
+    try {
+      await dbClient.connect();
+      await dbClient.runRawQuery(
+        `truncate ${[Object.values(databaseCollections).join(',')]} RESTART IDENTITY;`
+      );
+      await dbClient.disconnect();
+      process.exit(0);
+    } catch (err) {
+      console.log('Failed to truncate tables', err);
```

Comment:
You'll want `/* eslint-disable no-console */` at the top of this file to allow use of `console`. Otherwise, ESLint will throw warnings, and Huskey might even prevent commits from working.

Reply:
Also, shouldn't this be a console.error?

Reply:
Agreed, or at least warn.